app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,6 @@
+import dayjs from "dayjs";
 import * as R from "ramda";
 
 export const isPresent = R.pipe(R.either(R.isNil, R.isEmpty), R.not);
+
+export const calculateCreatedAgo = date => dayjs(date).fromNow();
```

Comment:
Why do we need to keep this util function in the global `src/utils`? Are we using it at more than one place right now? If not, move this to an `utils.js` within the closest scope.